src/Todo/Model.hs

```diff
@@ -7,18 +7,28 @@ module Todo.Model
     , Deadline (..)
     , Tags
     , Todos
+    , TodoDB (..)
```

Comment:
Let's create another module `Todo.DB` and move `TodoDB` type and all functions to work with `DB` in there. In future there will be much more functions so it's better to split logic now.